### DISCUSSION TITLE
How to specify the favorite icon of the page

### DISCUSSION THREAD

**Discussion by chenxizhang** [marp-team/marp#242] [Q&A] [answered]
How to define this like `title` and `description` directive?

**Comment by yhatt** [comment, marked as answer]
We have not yet provided to set custom path for favicon because [HTML output is not primary feature for us.](https://marp.app/docs#export-to-pdf-pptx-html)

General suggestion: you can set the icon by putting favicon to the root directory of your domain that is hosted Marp slide.

**Reply by chenxizhang** [reply]
Thanks to the quickly response,Yuki

And the suggestion is good for me, I think it will work for html output scenario.

**Reply by chenxizhang** [reply]
it works for me now. Thanks a lot for the great work.

![image](https://user-images.githubusercontent.com/1996954/146873870-a4a628d1-2bda-4c06-ae6f-f9d739bf4b5e.png)
